Formula/kontena-cli.rb

```diff
@@ -0,0 +1,185 @@
+  end
+
+  resource "semantic" do
+    url "https://rubygems.org/gems/semantic-1.6.0.gem"
+    sha256 "6ad9e301a564537f18ed01d0d346ab8ef54a908c0f7176937be940c34982804a"
+  end
+
+  resource "tty-color" do
+    url "https://rubygems.org/gems/tty-color-0.4.2.gem"
+    sha256 "2aa4da523710daf3e953f6c58145f5c9338feca76b0d2adfe25a4259cbc4b2c4"
+  end
+
+  resource "tty-cursor" do
+    url "https://rubygems.org/gems/tty-cursor-0.4.0.gem"
+    sha256 "4a08d4a7a89054bf5f388b630d921a186b6875f05854267422f45c5b5175901f"
+  end
+
+  resource "tty-prompt-0.12.0" do
+    url "https://rubygems.org/gems/tty-prompt-0.12.0.gem"
+    sha256 "60df198023dc7200b8f1484b1d6f2efe1da0181cb3a359ad88fdd507b1b74680"
+  end
+
+  resource "tty-screen" do
+    url "https://rubygems.org/gems/tty-screen-0.5.0.gem"
+    sha256 "c33a2d1d0b857cd37515a6db6fd5c6dd05e04f017b92753d3f31e54013bdb900"
+  end
+
+  resource "tty-table" do
+    url "https://rubygems.org/gems/tty-table-0.8.0.gem"
+    sha256 "87d81d2d1780de5beb8c25848b027ddcdec7fb407d766aafde0c4f032ccf36c8"
+  end
+
+  resource "unicode-display_width" do
+    url "https://rubygems.org/gems/unicode-display_width-1.1.3.gem"
+    sha256 "1d6247805b2431fd3d27614f6e0c38665b068bfc3a58cf89f08b7f107daf48e4"
+  end
+
+  resource "unicode_utils" do
+    url "https://rubygems.org/gems/unicode_utils-1.4.0.gem"
+    sha256 "b922d0cf2313b6b7136ada6645ce7154ffc86418ca07d53b058efe9eb72f2a40"
+  end
+
+  resource "verse" do
+    url "https://rubygems.org/gems/verse-0.5.0.gem"
+    sha256 "4c8af6322b91e115a772af1acdf99952e391ad14890577674e11e52d474af2e0"
+  end
+
+  resource "websocket-driver-kontena" do
+    url "https://rubygems.org/gems/websocket-driver-kontena-0.6.5.gem"
+    sha256 "7615511da99318dd8131de01587066416c36b7a630c9a2bd70ad3f987fbc2d3f"
+  end
+
+  resource "websocket-extensions" do
+    url "https://rubygems.org/gems/websocket-extensions-0.1.2.gem"
+    sha256 "7919c0310edda55ce4026d3f160e612160493c048276cabe500308439dd2161f"
+  end
+
+  resource "wisper" do
+    url "https://rubygems.org/gems/wisper-1.6.1.gem"
+    sha256 "5ae88a4aa8cab54ec849a934c4e70aebbef560114dd164dfd9b0058ec3077ca5"
+  end
+
+  resource "kontena-plugin-cloud" do
+    url "https://rubygems.org/gems/kontena-plugin-cloud-1.1.0.gem"
+    sha256 "87e88329c17c6aa6c49069fd6fa5f6577c451f73bb2bf264051b858af8ea57f1"
+  end
+
+  def install
+    ENV["GEM_HOME"] = libexec
+
+    cd "cli" do
+      system "gem", "build", "--norc", "kontena-cli.gemspec"
+
+      if build.head?
+        system "gem", "install", Dir["kontena-cli-*.gem"].first,
+         "--no-document", "--norc", "--install-dir", libexec
+      else
+        resources.each do |r|
+          r.verify_download_integrity(r.fetch)
+          system "gem", "install", r.cached_download, "--ignore-dependencies",
+                 "--no-document", "--norc", "--install-dir", libexec
+        end
+
+        system "gem", "install", "--ignore-dependencies",
```

Comment:
more args probably fit on this line. we're just trying to avoid going passed the eightieth column in terms of where to wrap the args